buildsystem/android_library.gradle

```diff
@@ -19,15 +19,24 @@ android {
     buildTypes {
 
         debug {
```

Comment:
these definitions should be directly in data's build.gradle. android_library.gradle is used by all android modules